redbot/cogs/trivia/trivia.py

```diff
@@ -437,7 +437,7 @@ def _get_leaderboard(self, data: dict, key: str, top: int):
 
         """
         channel = session.ctx.channel
-        LOG.debug("Ending trivia session; #%s in %s", channel, channel.guild.id)
+        LOG.debug(f"Ending trivia session; #{channel} in {channel.guild}")
```

Comment:
This (and any other log statements) should be reverted back to using % formatting. 

Unlike f-string usage, formatting is delayed for evaluation until the result is needed, and won't ever get formatted if the log level doesn't demand it.